python/example/accuracy_pitfalls.ipynb

```diff
@@ -0,0 +1,338 @@
+    "\n",
+    "\n",
+    "enable_features(\"contrib\")\n",
+    "enable_features(\"floating-point\")\n",
+    "\n",
+    "# establish data information\n",
+    "from opendp.trans import make_split_dataframe, make_cast_default, make_clamp, make_bounded_resize, \\\n",
+    "    make_sized_bounded_mean, make_select_column, make_impute_uniform_float, make_cast, make_cast_inherent\n",
+    "data_path = os.path.join('.', 'data', 'PUMS_california_demographics_1000', 'data.csv')\n",
+    "var_names = [\"age\", \"sex\", \"educ\", \"race\", \"income\", \"married\", \"pid\"]\n",
+    "D = pd.read_csv(data_path, names=var_names)\n",
+    "age = D.age\n",
+    "D_mean_age = np.mean(age)\n",
+    "age_lower_bound = 0.\n",
+    "age_upper_bound = 120.\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 49,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Accuracy interval (with accuracy value 0.2996) contains the true mean on D_tilde with probability 0.952\n",
+      "Accuracy interval (with accuracy value 0.2996) contains the true mean on D with probability 0.952\n"
+     ]
+    }
+   ],
+   "source": [
+    "# establish extra information for this simulation\n",
+    "age_bounds = (0., 100.)\n",
+    "data_size = 1000\n",
+    "n_sims = 1_000\n",
+    "epsilon = 1.\n",
+    "alpha = 0.05\n",
+    "releases = []\n",
+    "\n",
+    "D_tilde = np.clip(D.age, age_bounds[0], age_bounds[1])\n",
+    "D_tilde_mean_age = np.mean(D_tilde)\n",
+    "\n",
+    "def check_scale(scale, preprocessor, dataset_distance, epsilon):\n",
+    "    return (preprocessor >> make_base_laplace(scale)).check(dataset_distance, epsilon)\n",
+    "\n",
+    "preprocessor = (\n",
+    "    # Convert data into a dataframe of string columns\n",
+    "    make_split_dataframe(separator=\",\", col_names=var_names) >>\n",
+    "\n",
+    "    # Selects a column of df, Vec<str>\n",
+    "    make_select_column(key=\"age\", TOA=str) >>\n",
+    "\n",
+    "    # Cast the column as Vec<float>, and fill nulls with the default value, 0.\n",
+    "    make_cast_default(TIA=str, TOA=float) >>\n",
+    "\n",
+    "    # Clamp age values\n",
+    "    make_clamp(bounds=age_bounds) >>\n",
+    "\n",
+    "    # Resize the dataset to length `count_release`.\n",
+    "    #     If there are fewer than `count_release` rows in the data, fill with a constant of 20.\n",
+    "    #     If there are more than `count_release` rows in the data, only keep `count_release` rows\n",
+    "    make_bounded_resize(size=data_size, bounds=age_bounds, constant=20.) >>\n",
+    "\n",
+    "    # Compute the mean\n",
+    "    make_sized_bounded_mean(size=data_size, bounds=age_bounds)\n",
+    ")\n",
+    "\n",
+    "scale = binary_search(lambda s: check_scale(s, preprocessor, 1, epsilon), bounds=(0.0, 1000.0))\n",
```

Comment:
This works, but it is a bit more idiosyncratic to use 
`scale = binary_search_param(lambda s: preprocessor >> make_base_laplace(scale), 1, epsilon)`
And drop `def check_scale`